lib/git-whistles/pull_request/pivotal.rb

```diff
@@ -28,14 +28,16 @@ def tracker_related_params(team)
 
           log.info 'Finding your project and story¬'
 
-          PivotalTracker::Client.token = token
+          client = TrackerApi::Client.new(token: token)
           begin
-            story, project = PivotalTracker::Project.all.find do |project|
+            story, project = client.projects.find do |project|
               log.info '.¬'
-              story = project.stories.find(story_id) and break story, project
+              story = project.story(story_id) and break story, project
             end
             log.info '.'
-          rescue RestClient::Unauthorized
+          rescue StandardError => e
```

Comment:
What is the reason behind this change? Can't we rescue instead an additional rest client error code?

Reply:
The new gem doesn't use `RestClient` :/
I don't like this solution either, but this is what the new gem provides.

Reply:
Right, why not being explicit on rescuing `ClientError` then?

Reply:
hm, I must've tried this. Let me have a look and I'll tell you why not :)